CHANGELOG.md

```diff
@@ -4,7 +4,30 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
-## [Preview Release 2.0.0-preview4.20142.4 ] - 2020-05-21
+## [Stable Release 2.0.0] - 2020-06-16
+
+### Added
+- Added internal driver support to provide resiliency to DNS failures [#594](https://github.com/dotnet/SqlClient/pull/594)
+- Added support for `Active Directory Integrated`, `Active Directory Interactive` and `Active Directory Service Principal` authentication mode for .NET Core and .NET Standard [#560](https://github.com/dotnet/SqlClient/pull/560)
+- Added support for `Active Directory Service Principal` authentication mode for .NET Framework [#560](https://github.com/dotnet/SqlClient/pull/560)
+
+### Fixed
+- Fixed `SqlSequentialStream` multipacket read stalling issue in .NET Core [#603](https://github.com/dotnet/SqlClient/pull/603)
+- Fixed code page issue for Kazakh collation in SQL Server [#584](https://github.com/dotnet/SqlClient/pull/584)
+- Fixed stalled application issues when end of stream is reached [#577](https://github.com/dotnet/SqlClient/pull/577)
+- Fixed driver behavior to not throw exception for invalid configuration file [#573](https://github.com/dotnet/SqlClient/pull/573)
+- Fixed Object null reference issue when failover partner is set [#588](https://github.com/dotnet/SqlClient/pull/588)
+- Fixed `applicationintent` connection string property issue [#585](https://github.com/dotnet/SqlClient/pull/585)
+
+### Changes
+- Raise warning message when insecure TLS protocols are in use [#591](https://github.com/dotnet/SqlClient/pull/591)
+
+### Breaking Changes
+- Modified enclave provider interface `SqlColumnEncryptionEnclaveProvider` to be internal [#602](https://github.com/dotnet/SqlClient/pull/602) - _Since Secure enclaves is a relatively new feature, this change is not likely to impact customer applications_.
+- Updated `SqlClientMetaDataCollectionNames` exposed constants by removing non-existing constants and adding new to the metadata collection [#580](https://github.com/dotnet/SqlClient/pull/580)
+
```

Comment:
A lot of changes without any preview...

Reply:
The one for `SqlColumnEncryptionEnclaveProvider` should have happened when secure enclave support was implemented in 1.1. There was no need to leave it public like it had been in .NET Framework (it's extremely unlikely that anyone has implemented their own enclave provider) and there is a problem with the API which would have required an API change in the near future. So we decided to change it to internal on the major version change rather than have to bump the major version again very soon in order to fix it properly.

`SqlClientMetaDataCollectionNames` was broken as it was.